main.py

```diff
@@ -215,34 +216,34 @@ def buy():
 
                                 session[announcement_coin]['orders'].append(copy.deepcopy(order[announcement_coin]))
 
-                                logger.info(f"Partial fill order detected.  {order_status=} | {partial_amount=} out of {amount=} | {partial_fee=} | {price=}")
+                                LOG_INFO(f"Partial fill order detected.  {order_status=} | {partial_amount=} out of {amount=} | {partial_fee=} | {price=}")
                                 # FUTURE: We'll probably want to start attempting to sell in the future immediately after ordering any amount
                                 # It would require at least a minor refactor, since order is getting cleared and
                                 # it seems that this function depends on order being empty, but sell() depends on order not being empty.
                                 # globals.sell_ready.set()
 
                             # order not filled, try again.
-                            logger.info(f"Clearing order with a status of {order_status}.  Waiting for 'closed' status")
+                            LOG_INFO(f"Clearing order with a status of {order_status}.  Waiting for 'closed' status")
                             order.pop(announcement_coin)  # reset for next iteration
                 else:
-                    logger.warning(
+                    LOG_ERROR(
```

Comment:
I replaced this logger.warning with LOG_ERROR since the options.yaml state that available Levels are only: Debug, Info and Error (thats also why i commented out the respective LOG_WARNING, LOG_CRITICAL) .